.tool-versions

```diff
@@ -0,0 +1 @@
+ruby 3.2.1
```

Comment:
Let's switch from .tool-versions to .ruby-version since it's more compatible with other Ruby version managers. This will make it easier for people to contribute who might be using something other than asdf.

Reply:
Sounds good!